acceptable/make/Makefile.acceptable

```diff
@@ -0,0 +1,13 @@
+api-lint:
+	$(ENV)/bin/acceptable lint api.json $(ACCEPTABLE_MODULES) --quiet $(ARGS)
+
+api-update-metadata:
+	$(ENV)/bin/acceptable lint api.json $(ACCEPTABLE_MODULES) --quiet --update $(ARGS) 
+
+api-version:
+	$(ENV)/bin/acceptable api-version api.json $(ACCEPTABLE_MODULES)
+
+api-docs-markdown:
+	rm -f docs/en/*.md
+	$(ENV)/bin/acceptable render api.json --name "$(ACCEPTABLE_SERVICE_TITLE)" --dir docs
+
```

Comment:
Can we parameterise api.json and the docs/ dir too?

ACCEPTABLE_METADATA ?= api.json
ACCEPTABLE_DOCS ?= docs/

Also, mild preference to drop _TITLE, don't think its needed, ACCEPTABLE_SERVICE is good enough imo

Reply:
ACCEPTABLE_METADATA and  ACCEPTABLE_DOCS - yes
ACCEPTABLE_SERVICE_TITLE - Not sure, its called --name on the CLI but we use strings like "Some Service Name". So it seemed more like a title than a service name like "someservicename"??